src/main/java/org/tikv/common/TiConfiguration.java

```diff
@@ -360,12 +360,12 @@ private static ReplicaRead getReplicaRead(String key) {
   private int rawKVBatchWriteTimeoutInMS = getInt(TIKV_RAWKV_BATCH_WRITE_TIMEOUT_IN_MS);
   private int rawKVScanTimeoutInMS = getInt(TIKV_RAWKV_SCAN_TIMEOUT_IN_MS);
   private int rawKVCleanTimeoutInMS = getInt(TIKV_RAWKV_CLEAN_TIMEOUT_IN_MS);
-  private Optional<Integer> rawKVReadSlowLogInMS = getIntOption(TIKV_RAWKV_READ_SLOWLOG_IN_MS);
-  private Optional<Integer> rawKVWriteSlowLogInMS = getIntOption(TIKV_RAWKV_WRITE_SLOWLOG_IN_MS);
-  private Optional<Integer> rawKVBatchReadSlowLogInMS =
-      getIntOption(TIKV_RAWKV_BATCH_READ_SLOWLOG_IN_MS);
-  private Optional<Integer> rawKVBatchWriteSlowLogInMS =
-      getIntOption(TIKV_RAWKV_BATCH_WRITE_SLOWLOG_IN_MS);
+  private Integer rawKVReadSlowLogInMS = getIntOption(TIKV_RAWKV_READ_SLOWLOG_IN_MS).orElse(null);
```

Comment:
I'm a little confused. why not set the type to `int` and get the value via `getInt()` just like what we did for `rawKVCleanTimeoutInMS`?

@marsishandsome do you have any idea?

Reply:
I think `rawKVReadSlowLogInMS` not set default value in `org.tikv.common.TiConfiguration#loadFromDefaultProperties`, and rawKVReadSlowLogInMS must be have a default value due to `org.tikv.common.log.SlowLogImpl#log`.

Reply:
`rawKVReadSlowLogInMS` is calculated as follows:
1.  if `TIKV_RAWKV_READ_SLOWLOG_IN_MS` is set, then use it, in such case `rawKVReadSlowLogInMS` != null
2. else use `getTimeout() * 2`, in such case `rawKVReadSlowLogInMS` == null

Reply:
OK.